internal/moreos/testdata/fake_func-e.go

```diff
@@ -56,6 +56,10 @@ func main() {
 
 	// Block until we receive SIGINT or are canceled because Envoy has died.
 	<-sigCtx.Done()
+	stop()
```

Comment:
shouldn't this be called as `defer stop()` right after `signal.NotifyContext`?

Reply:
👍🏽